lib/unleash/strategy/email_domain.rb

```diff
@@ -0,0 +1,27 @@
+module Unleash
+  module Strategy
+    class EmailDomain < Base
```

Comment:
Can you explain the reasoning to not use the Incognia module?

Reply:
We need to define this way so `unleash` gem can properly load our strategies:

https://github.com/Unleash/unleash-client-ruby/blob/master/lib/unleash.rb#L8-L21